tests/test_relative_free_energy.py

```diff
@@ -187,6 +197,7 @@ def test_run_hif2a_test_system(estimate_relative_free_energy_fn):
         forcefield,
         n_frames=100,
         estimate_relative_free_energy_fn=estimate_relative_free_energy_fn,
+        sim_results_by_leg=sim_results_by_leg,
```

Comment:
I might be misunderstanding, but wasn't the issue in https://github.com/proteneer/timemachine/pull/983 that seeds were not consistent between runs in different processes? Would that issue be caught by this test?

(We do now have a consistency check between runs in separate processes in `test_examples.py`.)